AK/Variant.h

```diff
@@ -219,6 +219,8 @@ namespace AK {
 struct Empty {
 };
 
+constexpr bool operator==(Empty, Empty) { return true; }
```

Comment:
Does `=default` work as well?

Reply:
Not sure what is preferable here though...

Reply:
Sure - I'll try it

rebuild incoming :(